apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -317,6 +319,9 @@ export class DetailViewContents extends React.Component {
   };
 
   renderTopSection = () => {
+    const fit_workshop_link = (
```

Comment:
nit: should we use JS-style `fitWorkshopLink`? I don't care too strongly, esp since we're mixing styles in places for Ruby-supplied json keys. However those are usually object keys rather than local variables.